z3/tests/lib.rs

```diff
@@ -1128,6 +1128,30 @@ fn test_tactic_skip() {
     assert_eq!(goal_result.get_formulas::<Bool>(), vec![a.clone(), b, a],);
 }
 
+#[test]
+fn test_tactic_try_for() {
+    let cfg = Config::new();
+    let ctx = Context::new(&cfg);
+    let params = Params::new(&ctx);
+
+    let one = ast::Int::from_i64(&ctx, 1);
+    let two = ast::Int::from_i64(&ctx, 2);
+    let x = ast::Int::new_const(&ctx, "x");
+
+    let goal = Goal::new(&ctx, false, false, false);
+    goal.assert(&x.ge(&one.add(&two)));
+
+    let tactic = Tactic::new(&ctx, "simplify");
+    let try_for_tactic = tactic.try_for(4294967295);
```

Comment:
Please use the associated const [`u32::MAX`](https://doc.rust-lang.org/std/primitive.u32.html#associatedconstant.MAX) for this